docs/containers-registries.conf.5.md

```diff
@@ -7,33 +7,82 @@ containers-registries.conf - Syntax of System Registry Configuration File
 
 # DESCRIPTION
 The CONTAINERS-REGISTRIES configuration file is a system-wide configuration
-file for container image registries. The file format is TOML. The valid
-categories are: 'registries.search', 'registries.insecure', and
-'registries.block'.
+file for container image registries. The file format is TOML.
 
 By default, the configuration file is located at `/etc/containers/registries.conf`.
 
-# FORMAT
-The TOML_format is used to build a simple list format for registries under three
+# FORMATS
+
+## VERSION 2
+VERSION 2 is the latest format of the `registries.conf` and is currently in
+beta. This means in general VERSION 1 should be used in production environments
+for now.
+
+Every registry can have its own mirrors configured.  The mirrors will be tested
+in order for the availability of the remote manifest.  This happens currently
+only during an image pull.  If the manifest is not reachable due to connectivity
+issues or the unavailability of the remote manifest, then the next mirror will
+be tested instead.  If no mirror is configured or contains the manifest to be
+pulled, then the initially provided reference will be used as fallback.  It is
+possible to set the `insecure` option per mirror, too.
+
+Furthermore it is possible to specify a `prefix` for a registry.  The `prefix` is
+used to find the relevant target registry from where the image has to be pulled.
+During the test for the availability of the image, the prefixed location will be
+rewritten to the correct remote location.  This applies to mirrors as well as
+the fallback `location`.  If no prefix is specified, it defaults to the specified
+`location`.  For example, if `prefix = "example.com/foo"`, `location = "example.com"` and
+the image will be pulled from `example.com/foo/image`, then the resulting pull
+will be effectively point to `example.com/image`.  When a prefix is used, then a
+pull to the `location` without specifying the prefix will not be considered any more.
```

Comment:
Consider
```
[[registry]]
location = "physical.example.com"
prefix = "alias.example.com"
```

When there is no other interfering configuration, a pull from `physical.example.com` will work just fine without using the `alias.example.com` value of `prefix`; this seems to say that it “will not be considered”.

(We can defer this to the follow-up along with other documentation clean-ups.)

Reply:
Yeah I think I will remove this sentence, it's wrong.